Ports/SuperTuxKart/package.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=SuperTuxKart
+useconfigure='true'
+version='1.4'
+archive_hash='9890392419baf4715313f14d5ad60746f276eed36eb580636caf44e2532c0f03'
+files="https://github.com/supertuxkart/stk-code/releases/download/${version}/supertuxkart-${version}-src.tar.xz ${port}-${version}-src.tar.xz $archive_hash"
+auth_type='sha256'
+workdir="${port}-${version}-src"
+launcher_name='SuperTuxKart'
+launcher_category='Games'
+launcher_command='/usr/local/bin/supertuxkart'
+depends=(
+    'curl'
+    'freetype'
+    'glu'
+    'harfbuzz'
+    'libjpeg'
+    'libopenal'
+    'libogg'
+    'libpng'
+    'libvorbis'
+    'openssl'
+    'SDL2'
+    'sqlite'
+    'zlib'
+)
+
+configopts=(
+    '-DCMAKE_BUILD_TYPE=STKRelease'
+    "-DCMAKE_INSTALL_PREFIX=${SERENITY_INSTALL_ROOT}/opt/SuperTuxKart"
+    "-DSTK_INSTALL_DATA_DIR=${SERENITY_INSTALL_ROOT}/opt/SuperTuxKart"
```

Comment:
setting STK_INSTALL_DATA_DIR as an absolute path is the reason you have that awkward 5th patch.

Reply:
Setting `STK_INSTALL_DATA_DIR` to a relative path causes the final game to try the default location of `/home/domi/serenity/Build/x86_64/Root/opt/SuperTuxKart/opt/SuperTuxKart/data/`; if I change any of the other values it either doesn't build, because it tries to access `/opt/SuperTuxKart/data/` during install and fails, or compiles but ignores my path.

I think that STK isn't configured for being cross-built.

Reply:
Unfortunate, thanks for looking into it! If I had more time I might look into fixing those upstream 😅